README.md

```diff
@@ -49,9 +49,11 @@ client.on('ready', () => {
   console.log(`Logged in as ${client.user.tag}!`);
```

Comment:
```suggestion
  console.log(`Logged in as ${client.user.tag}!`);
  if (await client.application.commands.fetch().then(commands => commands.size) === 0) {
            await client.application.commands.set([
                {
                    name: "ping",
                    description: "Replies with Pong!"
                }
            ])
        }
```
I think example code should also handle the creation of the slash command, here is a suggestion.
Note: You have to make the ready listener async for this suggestion to work

Reply:
talked about this with crawl, i'll add the deploy code of the "advanced deployment" section of the guide